Messaging.md

```diff
@@ -379,15 +395,21 @@ See [the Oracle specifications](./Oracle.md#oracle-event) for more details.
 This type contains an `oracle_event` and a signature certifying its origination.
 See [the Oracle specifications](./Oracle.md#oracle-announcements) for more details.
 
-#### Version 0 `oracle_announcement`
+#### Version 1 `oracle_announcement`
 
-1. type: 55332 (`oracle_announcement`)
+1. type: 55354 (`oracle_announcement`)
 2. data:
-   * [`signature`:`annoucement_signature`]
-   * [`x_point`:`oracle_public_key`]
+   * [`signature`:`annoucement_pok_signature`]
+   * [`signature`:`attestation_pok_signature`]
+   * [`u16`:`nb_nonces`]
+   * [`nb_nonces*signature`:`nonce_pok_signatures`]
+   * [`x_point`:`oracle_announcement_public_key`]
+   * [`x_point`:`oracle_attestation_public_key`]
+   * [`nb_nonces*x_point`:`oracle_nonces`]
    * [`oracle_event`:`oracle_event`]
 
-where `signature` is a Schnorr signature over a sha256 hash of the serialized `oracle_event`, using the tag `announcement/v0`.
+where all `pok_signature`s are Schnorr signatures using the tag `announcement/v0` over a sha256 hash of the serialized `x_point`s concatenated in order and concatenated with `oracle_event`.
+That is to say, the signature hash is of the message `oracle_announcement_public_key||oracle_attestation_public_key||oracle_nonces||oracle_event`.
```

Comment:
My thoughts below :)

1. I think I've expressed this before -- sorry if I'm going over stuff that has already been discussed between you guys -- but I think the announcement should not have the oracle's public keys in it. They must be known beforehand. IIRC they are just being used as identifiers? I still think that is a dangerous choice but even in this case the `attestation_pok_signature` should not be in the announcement. The `attestation_pok_signature` should have been verified when the user originally saved the oracle's public key not at the time of any particular announcement.

2.  I think what was there before with `announcement_signature` made more sense because it's not a `pok` it's just a signature. Having all the oracle crypto data in `oracle_event` and having the signature over all of it in `oracle_announcement` made more sense to me.

3. Optional extra: IMO it's fine to do MuSig to aggregate all the `nonce_pok_signatures` into a single thing. Perhaps that is more specification work than you want to do now though. The reason I think it's fine is that the plan is to use MuSig to do cross-input aggregation which implies that it is secure for this purpose as well in a kind of *proof of possession* model which I think is reasonable here.

Reply:
Left the announcement key as an identifier, but removed attestation key and sig from announcement and moved it to a new `oracle_keys` message.